src/generate-font-faces.js

```diff
@@ -0,0 +1,20 @@
+export default function generateFontFaces(fontDictionary) {
+  const faces = [];
+  Object.keys(fontDictionary).forEach(fontName => {
+    const font = fontDictionary[fontName];
+    if (font) {
+      faces.push(
+        `@font-face {font-family: "${fontName}"; src: ${asFontFaceSrc(
+          font.url
+        )};}`
+      );
+    }
+  });
+  return '\n' + faces.join('\n');
+}
+
+function asFontFaceSrc(url) {
+  return Object.keys(url).map(
```

Comment:
What is the type of `url` here? Seeing `url` makes me think it should be a string, but we are calling `Object.keys` on it. I think some flow types in this repository would go a long way in making the code more readable

Reply:
Good point. It's a dictionary of font types (woff, woff2 etc) to url strings.
Will rename / comment for clarity

Reply:
Good point. It's a dictionary of font types (woff, woff2 etc) to url string.
Will rename / comment for clarity